crates/orchestrator/src/lib.rs

```diff
@@ -224,12 +225,17 @@ where
         // Calculate the bootnodes addr from the running nodes
         let mut bootnodes_addr: Vec<String> = vec![];
         for node in futures::future::try_join_all(spawning_tasks).await? {
+            let ip = node.inner.ip().await?;
             bootnodes_addr.push(
                 // TODO: we just use localhost for now
                 generators::generate_node_bootnode_addr(
                     &node.spec.peer_id,
-                    &LOCALHOST,
-                    node.spec.p2p_port.0,
+                    &ip,
+                    if ctx.ns.capabilities().requires_image {
+                        P2P_PORT
+                    } else {
+                        node.spec.p2p_port.0
+                    },
```

Comment:
Shouldnt be ```options.use_default_ports_in_cmd``` instead?

Reply:
We don't have access to those options here, is the same `if` we use to set the field.